### PR TITLE
New RWX licence info

### PR DESCRIPTION
**Pull request by cvlc**

### CODE REVIEW

File: docs/operations/licensing.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ You will need a license for Ondat if you want to make use of the full functional
...
-## Free Forever
+## Community Edition
 
-You can use Ondat for free without any time limit with a 1 TB usable capacity and up to 3 nodes. Please note that [ReadWriteMany (RWX)](/docs/concepts/rwx/) access mode Persistent Volumes is not available in this tier.
+Also known as 'Free Forever'. You can use Ondat with all features for free without any time limit with a 1 TB usable capacity and up to 3 nodes.
 
 ## Enterprise Premium
 
-You can obtain a tailor-made license suitable for your needs with premium features. For more information, contact hello@ondat.io or message us via Intercom.
+You can obtain a tailor-made license suitable for your needs with commercial support and a higher storage limit. For more information, contact hello@ondat.io or message us via Intercom.
```
Comment (cannischan) [inline docs/operations/licensing.md:24]: licence (to make sure it is consistent in this doc)

File: docs/operations/licensing.md [hunk 2]
```diff
@@ -13,15 +13,15 @@ You will need a license for Ondat if you want to make use of the full functional
...
 ## Free Trial
 
-You can use Ondat for free without any restrictions for 1 month.
+You can use Ondat with all features for free without any restrictions for 1 month.
 
-## Free Forever
+## Community Edition
 
-You can use Ondat for free without any time limit with a 1 TB usable capacity and up to 3 nodes. Please note that [ReadWriteMany (RWX)](/docs/concepts/rwx/) access mode Persistent Volumes is not available in this tier.
+Also known as 'Free Forever'. You can use Ondat with all features for free without any time limit with a 1 TB usable capacity and up to 3 nodes.
```
Comment (cannischan) [inline docs/operations/licensing.md:20]: Free Forever doesn't have rolling upgrades

**Review by cannischan — COMMENTED** [review]
Left some comments for small changes - thank you !

File: docs/operations/licensing.md (hunk 1 above)
Comment (cvlc) [reply docs/operations/licensing.md:24]:
```suggestion
You can obtain a tailor-made licence suitable for your needs with commercial support and a higher storage limit. For more information, contact hello@ondat.io or message us via Intercom.
```

**Comment by cvlc** [comment]
Thanks, @cannischan, I adjusted according to your feedback! 🚀

**Review by cannischan — APPROVED** [review]
LGTM !

**Pull request merged**